codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -53,6 +53,11 @@
 
         /**
          * Signal an error and release any retained buffers.
+         *<p>
+         * This method might be called multiple times.
```

Comment:
@nmittler @Scottmitch 

Maybe it's enough to give `complete()` a different name?. Basically we want to have the `error` method that gets called when something goes wrong and another method that gets called once the flow controller has passed the whole frame to Netty (currently `complete`).